src/app/pools/components/PoolForm/PoolForm.tsx

```diff
@@ -66,50 +65,46 @@ export const PoolForm = ({ pool, ...props }: Props): JSX.Element => {
     };
   }
 
-  useWindowTitle(title);
-
   return (
-    <FormCard sidebar={false} title={title}>
-      <FormikForm
-        aria-label={title}
-        cleanup={poolActions.cleanup}
-        errors={errors}
-        initialValues={initialValues}
-        onCancel={() => navigate({ pathname: urls.pools.index })}
-        onSaveAnalytics={{
-          action: "Saved",
-          category: "Resource pool",
-          label: "Add pool form",
-        }}
-        onSubmit={(values) => {
-          dispatch(poolActions.cleanup());
-          if (pool) {
-            dispatch(
-              poolActions.update({
-                ...values,
-                id: pool.id,
-              })
-            );
-          } else {
-            dispatch(poolActions.create(values));
-          }
-          setSaving(values.name);
-        }}
-        saved={saved}
-        savedRedirect={urls.pools.index}
-        saving={saving}
-        submitLabel={Labels.SubmitLabel}
-        validationSchema={PoolSchema}
-        {...props}
-      >
-        <FormikField label={Labels.PoolName} name="name" type="text" />
-        <FormikField
-          label={Labels.PoolDescription}
-          name="description"
-          type="text"
-        />
-      </FormikForm>
-    </FormCard>
+    <FormikForm
+      aria-label={title}
+      cleanup={poolActions.cleanup}
+      errors={errors}
+      initialValues={initialValues}
+      onCancel={() => navigate({ pathname: urls.pools.index })}
+      onSaveAnalytics={{
+        action: "Saved",
+        category: "Resource pool",
+        label: "Add pool form",
```

Comment:
`null removed successfully` appears after adding a new pool. This does not happen on main.

![Google Chrome screenshot 001292@2x](https://github.com/canonical/maas-ui/assets/7452681/35a59f9c-0454-4091-8ee6-2511d0751d9d)